README.md

```diff
@@ -98,12 +98,92 @@ profiles.create(
 ```ruby
 profiles.delete(customer_code: '02355E2e58Bf488EAB4EaFAD7083dB6A')
 # => {
-#       :code => 1,
-#       :message => "Operation Successful",
-#       :customer_code => "02355E2e58Bf488EAB4EaFAD7083dB6A",
+#      :code => 1,
+#      :message => "Operation Successful",
+#      :customer_code => "02355E2e58Bf488EAB4EaFAD7083dB6A",
 #    }
 ```
 
+### Payments
+
+*Summary*: Process payments using Credit Card, Payment Profile, Legato Token, Cash, Cheque, Interac, Apple Pay, or
+Android Pay.
+*Bambora Docs*: <https://dev.na.bambora.com/docs/references/payment_APIs/>
+
+To use the payments API, you must create an instance of the `Bambora::V1::PaymentResource` class.
+
+```ruby
+client = Bambora::JSONClient.new(...)
+payments = Bambora::V1::PaymentResource.new(client: client, sub_path: '/v1/payments')
+```
+
+Once the resource instance has been instantiated, actions can be made against the API.
+
+#### Make A Payment
+
+This is a lower level method that can be used to make a payment using any method. Below are some more specific and
+high level convenience methods.
+
+```ruby
+payments.make_payment(
+  {
+    amount: 50,
+    payment_method: 'card',
+    card: {
+      name: 'Hup Podling',
+      number: '4504481742333',
```

Comment:
I used some [test credit card](https://dev.na.bambora.com/docs/references/payment_APIs/test_cards/), which allow us to make test payments.